app-laptop/thinkfan/thinkfan-0.9.3.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Please bump to EAPI 6 here. Should be straightforward in that case with only minor changes